include/nuttx/timers/capture.h

```diff
@@ -40,6 +40,11 @@
 #define CAPIOC_FREQUENCE _CAPIOC(2)
 #define CAPIOC_EDGES     _CAPIOC(3)
 #define CAPIOC_ALL       _CAPIOC(4)
+#define CAPIOC_PULSES    _CAPIOC(5)
+#define CAPIOC_CLR_CNT   _CAPIOC(6)
```

Comment:
what's different from stop and then start?

Reply:
We can stop temporarily and start count again without losing count number.

Reply:
let's add the description, so other people can implement these new ioctl correctly.